src/node_internals.h

```diff
@@ -0,0 +1,90 @@
+// Copyright Joyent, Inc. and other Node contributors.

```

Comment:
None of the code in this file is (c) Joyent, it's all (c) yours truly.  You can leave out the copyright boilerplate or reduce it to just 'Node contributors.'

The same applies to the preceding file, I think: it's a mix of code authored by @mscdex and me (and maybe @indutny.)